### ISSUE TITLE
Cannot use 'discoverDomain' tool in online mode and with WLS 12.1.3

### ISSUE THREAD

**Issue by lukasged** [oracle/weblogic-deploy-tooling#96]
I'm running the deploy tooling version `0.10`, with an installation of WebLogic Server version `12.1.3` and JDK `1.8.0_162-b12`.

I'm trying to use the `discoverDomain` tool, in online mode (it seems that WLST script for WLS `12.1.3` have problems with `SingleSignOnServices` elements in offline mode only) but it fails with the following error:

```bash
$ /ORA/dbs01/oracle/product/Middleware/weblogic-deploy-tooling/weblogic-deploy/bin/discoverDomain.sh \
  -oracle_home /ORA/dbs01/oracle/product/Middleware \
  -domain_home /ORA/dbs01/oracle/product/Middleware/user_projects/domains/base_domain \
  -admin_url t3://localhost:8001 \
  -admin_user weblogic \
  -archive_file /tmp/DiscoveredWLSDomain.zip

JDK version is 1.8.0_162-b12
Domain type  not recognized by shell script...assuming JRF is required
JAVA_HOME = /ORA/dbs01/oracle/product/jdk8
WLST_EXT_CLASSPATH =
CLASSPATH = /ORA/dbs01/oracle/product/Middleware/weblogic-deploy-tooling/weblogic-deploy/lib/weblogic-deploy-core.jar
WLST_PROPERTIES = -Djava.util.logging.config.class=oracle.weblogic.deploy.logging.WLSDeployLoggingConfig -Dcom.oracle.cie.script.throwException=true
/ORA/dbs01/oracle/product/Middleware/wlserver/common/bin/wlst.sh /ORA/dbs01/oracle/product/Middleware/weblogic-deploy-tooling/weblogic-deploy/lib/python/discover.py -oracle_home /ORA/dbs01/oracle/product/Middleware -domain_home /ORA/dbs01/oracle/product/Middleware/user_projects/domains/base_domain -admin_url t3://localhost:8001 -admin_user weblogic -archive_file /tmp/DiscoveredWLSDomain.zip
Java HotSpot(TM) 64-Bit Server VM warning: ignoring option MaxPermSize=256m; support was removed in 8.0
The discoverDomain program will write its log to /ORA/dbs01/oracle/product/Middleware/weblogic-deploy-tooling/weblogic-deploy/logs/discoverDomain.log

Initializing WebLogic Scripting Tool (WLST) ...

Welcome to WebLogic Server Administration Scripting Shell

Type help() for help on available commands

Problem invoking WLST - Traceback (innermost last):
  File "/ORA/dbs01/oracle/product/Middleware/weblogic-deploy-tooling/weblogic-deploy/lib/python/discover.py", line 452, in ?
  File "/ORA/dbs01/oracle/product/Middleware/weblogic-deploy-tooling/weblogic-deploy/lib/python/discover.py", line 412, in main
  File "/ORA/dbs01/oracle/product/Middleware/weblogic-deploy-tooling/weblogic-deploy/lib/python/discover.py", line 81, in __process_args
  File "/ORA/dbs01/oracle/product/Middleware/weblogic-deploy-tooling/weblogic-deploy/lib/python/discover.py", line 130, in __process_online_args
  File "/ORA/dbs01/oracle/product/Middleware_12.1.3.0_generic/weblogic-deploy-tooling/weblogic-deploy/lib/python/wlsdeploy/util/getcreds.py", line 28, in getpass
        at java.util.ResourceBundle.getObject(ResourceBundle.java:450)
        at java.util.ResourceBundle.getString(ResourceBundle.java:407)
        at oracle.weblogic.deploy.exception.ExceptionHelper.getMessage(ExceptionHelper.java:41)
        at oracle.weblogic.deploy.util.CLAUtils.promptInTerminalWindow(CLAUtils.java:76)
        at oracle.weblogic.deploy.util.CLAUtils.getPasswordInput(CLAUtils.java:64)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)

java.util.MissingResourceException: java.util.MissingResourceException: Can't find resource for bundle java.util.PropertyResourceBundle, key WLSDPLY-09045


discoverDomain.sh completed but with some issues (exit code = 1)
```

**Comment by rakillen** [comment]
The discoverDomain tool does not currently support online discovery.  There is a missing message resource here that would have helped identify the problem.  We are fixing that message, and looking into the SingleSignOnServices issue.

**Comment by rakillen** [comment]
The missing message resources have been corrected and merged, and will appear in the next release.

**Comment by lukasged** [comment]
OK, thanks @rakillen !

**Comment by ddsharpe** [comment]
Adding online support for Discover has been in our backlog of TODOs.  Adding this feature to the release schedule.

**Comment by CarolynRountree** [comment]
Closed via #126
